scripts/package-build/aws-gwlbtun/build.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
```

Comment:
Do you think it's possible to call the script from a reusable location? I really would like to avoid duplicating it, if that's at all possible.

Reply:
Let me try it and then see what it will be.
At least some of them could be reusable.

Reply:
@dmbaturin @c-po PR ready for review
Most of the packages a built by symlinked to  general `build.py`
Only these packages use their own `build.py`
```
aws-gwlbtun
linux-kernel
netfilter
```